arangod/RocksDBEngine/RocksDBCollection.cpp

```diff
@@ -2057,13 +2058,11 @@ void RocksDBCollection::destroyCache() const {
   }
   TRI_ASSERT(CacheManagerFeature::MANAGER != nullptr);
   // must have a cache...
-  TRI_ASSERT(_cacheEnabled);
-  TRI_ASSERT(_cachePresent);
+  TRI_ASSERT(_cacheEnabled || _cachePresent);
```

Comment:
This should be `&&`, right?

Reply:
This doesn’t  quite work with the updateProperties method